lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -75,7 +75,7 @@ def get_dir_vector(zdir):
 
 def _viewlim_mask(xs, ys, zs, axes):
```

Comment:
Should xs, ys, zs be aggregated into one array? You're almost always unpacking some array into the arguments.
Likely doesn't matter much performance-wise, because the unpacking only creates views and you'd otherwise create them internally by sliching for the viewlim comparisons.

Reply:
I don't think it matters too much since it's operating on views like you say. There's one place where the coordinates are stored separately and IMO the function here is a lot cleaner with the arguments split already, so would lean keeping as-is if that's good with you.